Packaging/Package.appxmanifest

```diff
@@ -301,6 +301,45 @@
         </uap:Extension>
       </Extensions>
     </Application>
+    <Application Id="DevHome.HostsFileEditor" Executable="DevHome.HostsFileEditor\DevHome.HostsFileEditor.exe" EntryPoint="Windows.FullTrustApplication">
+      <uap:VisualElements DisplayName="ms-resource:HostsFileEditorAppDisplayName" Description="ms-resource:HostsFileEditorAppDescription" BackgroundColor="transparent" Square150x150Logo="Assets\HostsFileEditor\Square150x150Logo.png" Square44x44Logo="Assets\HostsFileEditor\Square44x44Logo.png">
+        <uap:DefaultTile Wide310x150Logo="Assets\HostsFileEditor\Wide310x150Logo.png" />
+        <uap:SplashScreen Image="Assets\HostsFileEditor\SplashScreen.png" />
+      </uap:VisualElements>
+      <Extensions>
+        <uap3:Extension Category="windows.appExecutionAlias" EntryPoint="Windows.FullTrustApplication">
+          <uap3:AppExecutionAlias>
+            <desktop:ExecutionAlias Alias="DevHome.HostsFileEditorApp.exe" />

```

Comment:
These are very verbose app execution aliases. Have we thought about something shorter?

Reply:
Yeah, I wanted to keep it explicit. I can make them like DevHome.Hosts, DevHome.RegPreview and DevHome.EnvVariables.